feature_extraction/chi_squared.ipynb

```diff
@@ -0,0 +1,863 @@
+{
```

Comment:
<p>I belive read per millions are never negative. Also the data wasn't normalized before the fold change so that may cause a discrepancy </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/treyduffus/multiclass_lungcancer_classification_models/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
fixed! added MinMaxScaler for proper normalization and ensuring non-negative values.